platformLibs/src/platform/ios/UIKit.def

```diff
@@ -5,3 +5,4 @@ modules = UIKit
 
 compilerOpts = -framework UIKit
 linkerOpts = -framework UIKit
+excludedFunctions = UISceneErrorDomain
```

Comment:
I can't find this function in `UIKIt`. Where is it located?

Reply:
I see, `excludedFunctions` filters globals too.

Reply:
`UIKit.framework/Headers/UISceneDefinitions.h`